README.md

```diff
@@ -3,13 +3,36 @@
 [![All Contributors](https://img.shields.io/badge/all_contributors-3-orange.svg?style=flat-square)](#contributors-)
 <!-- ALL-CONTRIBUTORS-BADGE:END -->
 
-Mautic Introduction
-===========
+[![codecov](https://codecov.io/gh/mautic/mautic/branch/features/graph/badge.svg)](https://codecov.io/gh/mautic/mautic)
+<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+[![All Contributors](https://img.shields.io/badge/all_contributors-3-orange.svg?style=flat-square)](#contributors-)
+<!-- ALL-CONTRIBUTORS-BADGE:END -->
+
+
+About Mautic
+============
+Mautic is the world’s largest open-source marketing automation project. With over 200,000 organisations using Mautic and over 1,000 community volunteers, we empower businesses by making it easy to manage their marketing across a range of channels. Stay up to date about initiatives, releases and strategy via our [blog](https://www.mautic.org/blog).
+
+Marketing automation has historically been difficult to implement within organisations. The Mautic community is an example of open source at its best, offering great software and a vibrant and caring community in which to learn and share knowledge.
+
+Open source means more than open code. Open source provides equality for all and a chance for everyone to improve.
+
 ![Mautic](.github/readme_image.png "Mautic Open Source Marketing Automation")
 
+Get Involved
+=============
+Before we tell you how to install and use Mautic, we like to shamelessly plug our awesome user and developer communities! Users, start [here](https://www.mautic.org/community/get-involved) for inspiration, or follow us on Twitter [@MauticCommunity](https://twitter.com/MauticCommunity) or Facebook [@MauticCommunity](https://www.facebook.com/MauticCommunity/). And once you’re familiar with using the software, maybe you will share your wisdom with others in our [Slack](https://www.mautic.org/community/get-involved/communication-channels) channel.
+
+Calling all devs, testers and tech writers! Technical contributions are also welcome. First, read our [general guidelines](https://contribute.mautic.org/contributing-to-mautic) about contributing. If you want to contribute code, read our [CONTRIBUTING.md](https://github.com/mautic/mautic/blob/feature/.github/CONTRIBUTING.md) or [Contributing Code](https://contribute.mautic.org/contributing-to-mautic/developer) docs then check out the issues with the [L1 label](https://github.com/mautic/mautic/issues?q=is%3Aissue+is%3Aopen+label%3AL1) to get stuck in quickly and show us what you’re made of.
```

Comment:
Refactoring to use link macros and updating to T1 rather than L1 - goodness this was out of date! 😊 

```suggestion
Calling all devs, testers and tech writers! Technical contributions are also welcome. First, read our [general guidelines][contributing] about contributing. If you want to contribute code, read our [CONTRIBUTING.md][contributing-md] or [Contributing Code][contribute-developer] docs then check out the issues with the [T1 label][t1-isssues] to get stuck in quickly and show us what you’re made of.
```